Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -353,9 +353,7 @@ extension TopBannerViewController: NavigationComponent {
         instructionsBannerView.update(for: instruction)
         lanesView.update(for: instruction)
         nextBannerView.navigationService(service, didPassVisualInstructionPoint: instruction, routeProgress: routeProgress)
-        DispatchQueue.main.async {
```

Comment:
Is it guaranteed to be called on the main queue?

Reply:
Other views are modified here. So I would say yes. We will have a main thread sanitizer yelling at us if it won't be.